contracts/PartnerConfiguration/PartnerConfiguration.sol

```diff
@@ -161,17 +180,32 @@ contract PartnerConfiguration is IPartnerConfiguration, Ownable {
         uint256 /* expires */,
         uint256 duration
     ) external view override returns (uint256) {
-        require(
-            (duration >= 1) && (duration >= _minDuration),
-            "PartnerConfiguration: Less than min duration"
-        );
-
-        if ((_maxDuration != 0) && (duration > _maxDuration))
-            revert("PartnerConfiguration: More than max duration");
-
         if (duration == 1) return 2 * (10 ** 18);
         if (duration == 2) return 4 * (10 ** 18);
 
         return (duration + 2) * (10 ** 18);
     }
+
+    /**
+       @inheritdoc IPartnerConfiguration
+     */
+    function isValidName(string calldata name, uint256 duration) external view {
```

Comment:
if it is not going to return a bool, wouldn't it be better to call it "ValidateName"?

Reply:
I agree, I was going to raise it but wanted to follow what we agreed initially. Will change the name.
Thank you.